src/summary/summary.ts

```diff
@@ -167,5 +168,9 @@ export class Summary {
       dataTypes: Array.from(this.dataTypes),
       schemaVersion: this.schemaVersion,
     }
+    if (this.schemaSource) {
+      output.schemaSource = this.schemaSource
+    }
```

Comment:
The nice thing about `undefined` is that it acts like an absence, so you can just pass it through and have the same effect:

```suggestion
      schemaVersion: this.schemaVersion,
      schemaSource: this.schemaSource,
    }
```